site/content/en/docs/Guides/feature-stages.md

```diff
@@ -28,6 +28,7 @@ The current set of `alpha` and `beta` feature gates are:
 |--------------|---------|---------|-------|-------|
 | Multicluster Allocation<sup>*</sup> | N/A | Enabled | `Alpha` | 0.11.0 |
 | Example Gate (not in use) | `Example` | Disabled | None | 0.13.0 |
+| [Port Allocations to Multiple Containers]({{< ref "/docs/Reference/gameserver.md" >}}) | `ContainerPortAllocation` | Disabled | `Alpha` | 1.5.0 |
```

Comment:
You'll hate me, but this need to be wrapped in a `feature` shortcode like you did previously, so it doesn't show up with this release. :man_facepalming:  I should have mentioned that in my comments previously.

(Might need to make a copy of the table, not sure, depends how formatting works out)

I swear this is the last thing :+1:

Reply:
Good point! I had to wrap the table as you mentioned but all done now :)